### ISSUE TITLE
🐛 "cloudflared tunnel route dns" uses incorrect domain name

### ISSUE THREAD

**Issue by ysmood** [cloudflare/cloudflared#797]
I have two domains, one is a.com another is b.com.

I want to create a dns route for `test.a.com`, but it it will instead create a route for `test.a.com.b.com`.

Here's the command to reproduce it:

```bash
$ cloudflared tunnel route dns ys-pc test.a.com             
2022-10-31T05:56:58Z INF Added CNAME test.a.com.b.com which will route to this tunnel tunnelID=4885d9e3-331e-4a45-b0c0-af6c3e687530
```

**Comment by anacrolix** [comment]
Is there a workaround for this?

**Comment by pestevao** [comment]
I think you have to make an additional tunnel for domain a.com.
Seems that you only have one active on the b.com domain.

**Comment by zhaoruibing** [comment]
Duplicate with #905 ?

Though the [documentation](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/configure-tunnels/local-management/configuration-file/) is misleading and perhaps could be more clear?
It provides an example config file for a locally-managed tunnel, where it allows two different domains.

```yml
tunnel: 6ff42ae2-765d-4adf-8112-31c55c1551ef
credentials-file: /root/.cloudflared/6ff42ae2-765d-4adf-8112-31c55c1551ef.json
originRequest: # Top-level configuration
  connectTimeout: 30s

ingress:
  # The localhost:8000 service inherits all root-level configuration.
  # In other words, it will use a connectTimeout of 30 seconds.
  - hostname: example.com
    service: localhost:8000
  - hostname: example2.com
    service: localhost:8001
```
